olp-cpp-sdk-core/include/olp/core/client/OlpClientSettings.h

```diff
@@ -151,6 +152,16 @@ struct RetrySettings {
    */
   using BackdownPolicy = std::function<int(int)>;
 
+  /**
+   * @brief A strategy which takes the initial backdown period in milliseconds
+   * with current retry count, and returns what should be used for the next wait
+   * timeout.
+   *
+   * Is superior to the \ref BackdownPolicy, as it computes wait time based on
+   * the retry count as well.
```

Comment:
This note is not related to the type. I'd move it closer to the setting itself (i.e. `backdown_strategy`).